docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -249,22 +249,23 @@ and provider discovery. The only supported flow is the authorization code flow w
-| authentication.tokenCookie.name      | auxiliary   | string         |                             | name of cookie to store access token                                                                          |
-| authentication.tokenCookie.path      | auxiliary   | string         |                             | path of access token cookie                                                                                   |
-| authentication.tokenCookie.domain    | auxiliary   | string         |                             | domain of access token cookie                                                                                 |
+| authentication.redirectUri           | optional    | url            | inferred from UI's location | Callback URL to which a user is redirected after successful authentication                                                  |
+| authentication.scope                 | optional    | string         | `openid profile`            | Scope parameter's value sent to the authorization endpoint.                                                                 |
+| authentication.authorizationEndpoint | auxiliary   | url or path    | discovered                  | Absolute URL or path relative to `Issuer` overriding the value retrieved from the OpenID Provider                           |
+| authentication.tokenEndpoint         | auxiliary   | url or path    | discovered                  | as above                                                                                                                    |
+| authentication.userinfoEndpoint      | auxiliary   | url or path    | discovered                  | as above                                                                                                                    |
+| authentication.jwksUri               | auxiliary   | url or path    | discovered                  | as above                                                                                                                    |
+| authentication.tokenCookie.name      | auxiliary   | string         |                             | name of cookie to store access token                                                                                        |
+| authentication.tokenCookie.path      | auxiliary   | string         |                             | path of access token cookie                                                                                                 |
+| authentication.tokenCookie.domain    | auxiliary   | string         |                             | domain of access token cookie                                                                                               |
+| authentication.accessTokenIsJwt      | optional    | boolean        | true                        | Access token might be of `JWT` format but it's not mandated by the `OIDC` spec                                              |
```

Comment:
I'm missing some part in this configuration option name or description. Access token might be either a `JWT` or ...? From looking into the code it looks like than it is a Bearer token that is used for obtaining the `JWT` by calling `userinfoEndpoint` . Am I right? Maybe we should call it `accessTokenPurpose` with options: `JWT | UserInfoBearerToken` or sth like this? Or at least we could improve the description